vita3k/renderer/src/creation.cpp

```diff
@@ -251,18 +251,23 @@ void destroy(SceGxmSyncObject *sync, State &state) {
     // nothing to do right now
 }
 
-bool init(SDL_Window *window, std::unique_ptr<State> &state, Backend backend, const Config &config, const char *base_path) {
+bool init(SDL_Window *window, std::unique_ptr<State> &state, Backend backend, const Config &config, const Root &root_paths) {
     switch (backend) {
     case Backend::OpenGL:
         state = std::make_unique<gl::GLState>();
-        if (!gl::create(window, state, base_path, config.hashless_texture_cache))
+        state->cache_path = root_paths.get_pref_path_string();
+        state->log_path = root_paths.get_log_path_string();
+        state->shared_path = root_paths.get_shared_path_string();
+        if (!gl::create(window, state, config))
             return false;
         break;
 
     case Backend::Vulkan:
         state = std::make_unique<vulkan::VKState>(config.gpu_idx);
-
-        if (!vulkan::create(window, state, base_path, config))
+        state->cache_path = root_paths.get_pref_path_string();
```

Comment:
Cache path is not a pref path. Same for OpenGL

Reply:
Fixed.